composer.json

```diff
@@ -7,9 +7,9 @@
     "php": ">=7.1",
     "spryker-shop/shop-application": "^1.0.0",
     "spryker-shop/shop-ui": "^1.6.0",
-    "spryker/availability": "^5.4.0 || ^6.0.0",
+    "spryker/availability": "^5.4.0 || ^6.0.0 || ^8.0.0",
```

Comment:
Composer --prefer-lowest:

Updating spryker/availability (8.0.0 => 6.0.0): Loading from cache

I think we should remove 5.4.0 here.

Reply:
Please do so, and approve then

Reply:
Actually, not true.
We still have the "bad" one in master (latest release) - https://github.com/spryker-shop/cart-page/blob/master/composer.json#L10
As such you cant remove it here either.

We should be doing a better job in general in terms of BC promise. But thats a different story.
So this is then good to go IMO.

Reply:
Let's avoid using the cumulative effects of all modules installed and stay based on module dependencies.